File: src/test/java/com/jason/StudentTest.java
package com.jason;

import com.jason.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

    @Test
    void highestScoreTest() {
        Student student = new Student("Jason", 88, 99);
        Assertions.assertEquals(99, student.highest());
    }

    @Test
    void averageTest() {
        Student student = new Student("JJ", 90, 80);
        Assertions.assertEquals((90+80)/2, student.getAverage());
    }
}

File: src/test/kotlin/com/jason/StudentTest.kt
package com.jason.kotlin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {

    @Test
    fun highestTest() {
        val student = Student("Jason", 90, 70)
        Assertions.assertEquals(90, student.highest())
    }

    @Test
    fun averageTest() {
        val student = Student("Jason", 90, 70)
        Assertions.assertEquals((70+90)/2, student.getAverage())
    }

    @Test
    fun gradingTest() {
        val student = Student("Jason", 90, 70)
        Assertions.assertEquals('B', student.grading())
    }
}
